src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +11,15 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] tempArray = text.replaceAll("[^-0-9]+", " ").split(" ");
+        text = text.join(" ", tempArray).trim();
```

Comment:
split and then join and one more time split. Seems redundant  to me. Try to simplify it